src/styles/colors.scss

```diff
@@ -1,2 +1,6 @@
 $access-green: #1f9614;
 $off-white: #FFFFFE;
+$onelogin-blue: #00A9E0;
+$onelogin-black: #1C1F2A;
+$onelogin-gray: rgb(243, 243, 244);
```

Comment:
Did you want to use #F3F3F4 instead of rgb(243, 243, 244)

If you're interested in how they're the same https://www.rapidtables.com/convert/number/how-decimal-to-hex.html

Reply:
I have been using exactly what was in the onelogin marketing manual, but this makes sense - keep it with same "color - group" :-) ((https://www.rgbtohex.net/))

Reply:
Generally it doesn't matter a whole lot. I typically won't use rgb unless I need to specify opacity or something where I cant get away with hex value. Good to go for consistency though. 😄